com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/util/TextureUtil.java

```diff
@@ -214,8 +217,19 @@ public static TextureProfile getTextureProfileByPath(TextureProfiles textureProf
         return null;
     }
 
-    public static TextureImage createCombinedTextureImage(TextureImage[] textures, TextureImage.Type type) throws TextureGeneratorException {
-        int numTextures = textures.length;
+    private static int getTextureGenerateResultByteIndex(TextureImage textureImage, int alternative, int mipMap) throws TextureGeneratorException {
```

Comment:
The method name is a bit confusing. `ByteIndex` sounds like it refers to an offset in a continuous array of bytes, but it is really the index of one of the byte arrays inside the `GenerateResult.imageDatas`.